cmake/external_tbb_prebuilt.cmake

```diff
@@ -37,11 +37,15 @@ ExternalProject_Add(
 ExternalProject_Get_Property(ext_tbb SOURCE_DIR)
 set(SOURCE_DIR ${SOURCE_DIR}/${ARCHIVE_FILE_BASE})
 
-set(TBB_LINK_LIBS
-    ${SOURCE_DIR}/lib/${CMAKE_SHARED_LIBRARY_PREFIX}clangTooling${CMAKE_SHARED_LIBRARY_SUFFIX}
-    ${SOURCE_DIR}/lib/${CMAKE_SHARED_LIBRARY_PREFIX}clangTooling${CMAKE_SHARED_LIBRARY_SUFFIX}
-    ${SOURCE_DIR}/lib/${CMAKE_SHARED_LIBRARY_PREFIX}clangTooling${CMAKE_SHARED_LIBRARY_SUFFIX}
-)
+if (WIN32)
+    set(TBB_LINK_LIBS ${SOURCE_DIR}/lib/intel64/vc14/tbb.lib)
+else()
+    set(TBB_LINK_LIBS
```

Comment:
nit: this `else` could be deleted because external_tbb_prebuilt.cmake is only invoked on Windows.

Reply:
I would like to keep it there. Currently it is only used on Windows, but later it may be used on other system. It is always better to make codes portable.